testsuite/features/build_validation/add_custom_repositories/add_ceos8_repositories.feature

```diff
@@ -49,15 +49,16 @@ Feature: Adding the CentOS 8 distribution custom repositories
     When I wait until the channel "centos-8-iso" has been synced
 
   Scenario: Create CLM filters to remove AppStream metadata
+    Given I am authorized for the "Admin" section
```

Comment:
This step is completely unnecessary :)
We follow an element from the left menu, those are always accessible, and we are already logged in.
Remember that since a few months ago we keep the browser session between scenarios, so we keep it during the whole Cucumber feature.

Reply:
OK I will remove it, but the failure screenshot was showing to me a login screen....

Reply:
Done.

Reply:
After our discussion, I will re-add it.

Reply:
Yes, let's have this line until we decided if we tweak the session timeout during the deployment.